README.md

```diff
@@ -143,10 +142,17 @@ docker compose run --rm django python manage.py graph_models -a -o models.dot --
 dot -Tsvg  models.dot > models.svg
 ```
 
+### ML Backend
+
+Add the minio hostname to /etc/hosts.
```

Comment:
Move this to the Data Storage section, since its relevant to the whole project not just the ML backend